docs/OnlineTutorial/guide.md

```diff
@@ -1443,19 +1443,19 @@ in the unnecessary annotation for the fib function:
 
 ``` {.editonly}
 function fib(n: nat): nat
-   decreases n
+  decreases n
 {
-   if n == 0 then 0 else
-   if n == 1 then 1 else
-                  fib(n - 1) + fib(n - 2)
+  if n == 0 then 0 else
+  if n == 1 then 1 else
+                 fib(n - 1) + fib(n - 2)
```

Comment:
```suggestion
  if n == 0 then 0 else
  if n == 1 then 1 else
  fib(n - 1) + fib(n - 2)
```

Reply:
Same comment - the value should be indented. My style would be to put the else on the next line indented the same as the 'then'